### ISSUE TITLE
segment called as homozygous deletions despite heterozygous SNPs

### ISSUE THREAD

**Issue by sa256** [mskcc/facets#98]
Hello,

I have results from facets where few regions have been called as homozygous deletions (tcn.em=0 and lcm.em=0), despite presence of heterozygous sites. Following are few such entries (complete output is also attached herewith):

*15	67	1211	113	-0.147	-0.147	11	-0.156	-0.069	19887721	40220705	0.13	0	0
*16	73	968	192	-0.26	0.0011	7	-0.239 0.0011	62595	799456	0.135	0	0
*16	75	1745	131	-0.356	-0.053	5	-0.35	-0.031	4797316	28585795	0.251420	0	0

Could you please suggest why are these called as deletions?
[out.txt](https://github.com/mskcc/facets/files/2394227/out.txt)

**Comment by veseshan** [comment]
You are right it shouldn't be called homozygous deletions, but not because there are het snps since those comes from the normal cells in the sample. These are fairly large segments that cannot be homozygously deleted. Something is odd with the fit since a large number of segments are assigned tcn=0 and hardly any tcn=1

> table(xx$tcn.em)

 0  1  2  4  5  6  7  8 11 12 13 14 15 16 18 19 
39  4 52 11  2  1  5  3  4  2  1  1  1  2  1  1 

Does the naive (straight from procSample) fit have the same problem? If you can share the count matrix it will help debug. You can remove the genomic position and allele information columns if privacy is an issue.

Venkat

**Comment by veseshan** [comment]
You have attached the out matrix from procSample which shows that the naive fit is also wrong. I can't debug without the count matrix i.e. the jointseg component of procSample output.

**Comment by sa256** [comment]
I apologize for the wrong files. Please find the attached count matrices with filters (jointSeg_with_filters.txt) and without filters (jointSeg_wo_filters.txt) for snp-pileup.
[jointSeg_with_filters.txt](https://github.com/mskcc/facets/files/2402980/jointSeg_with_filters.txt)
[jointSeg_wo_filters.txt](https://github.com/mskcc/facets/files/2402981/jointSeg_wo_filters.txt)

**Comment by veseshan** [comment]
Something is wrong with your data files. From the out matrix you had included up top
```
> xx <- read.table("out.txt", header=T)
> sum(xx$num.mark)
[1] 193823
```
However there are far fewer lines in the files. Oddly with filter has more lines than wo.
```
$ wc jointSeg_w*
  57542  920672 5083708 jointSeg_with_filters.txt
  40056  640896 3683372 jointSeg_wo_filters.txt
  97598 1561568 8767080 total
```

**Comment by sa256** [comment]
Hello Venkat, please find these revised files:
1) fit$cncf (b1t1_snp-pileup_withFilters.csv.gz_facets.txt)
2)  oo$jointSeg i.e count matrix (b1t1_snp-pileup_withFilters.csv.gz_facets_jointseg.txt)

[b1t1_snp-pileup_withFilters.csv.gz_facets_jointseg.txt](https://github.com/mskcc/facets/files/2405985/b1t1_snp-pileup_withFilters.csv.gz_facets_jointseg.txt)
[b1t1_snp-pileup_withFilters.csv.gz_facets.txt](https://github.com/mskcc/facets/files/2405986/b1t1_snp-pileup_withFilters.csv.gz_facets.txt)

I apologize for the confusion before.

**Comment by sa256** [comment]
Hello Venkat,

I could try different depth parameters and update you with new calls. but I have a question:

Actually we have done multi region sequencing of same tumor sample and trying to call cnvs from each region. The data I shared with you was from one region of one sample. I also have calls made form two more regions from the same tumor. The calls with tcn=0 are very less in other two regions as compared to the one I shared with you (with similar coverages). What is the possibility that one of the regions has actually acquired more copy number alterations as compared to other regions i.e these are true calls instead of spurious calls because of low coverage? It would be great if you could share your views about it. 

I am attaching a text file with coverages of the tumor samples (total 6 samples with 3-5 regions from each sample), their corresponding normal samples, percentage of tcn=0 calls and total loci called in jointseg matrix.

Thanks

[stats_of_all_samples.txt](https://github.com/mskcc/facets/files/2411268/stats_of_all_samples.txt)

**Comment by veseshan** [comment]
I haven't worked on multi-region samples and not getting a good sense of it.
